config/sync/block.block.sitesearchblock.yml

```diff
@@ -0,0 +1,26 @@
+uuid: bcd31f12-3b6f-4852-be60-03e09b9a8818
+langcode: en
+status: true
+dependencies:
+  module:
+    - nidirect_common
+    - system
+  theme:
+    - bartik
```

Comment:
Should be nidirect, or can this not be tested with in its current state?

Reply:
Not yet, it doesn't give a very helpful presentation. Once it's fleshed out more I'm happy to move this into the right place.

@David-Corbett if you're already looking at page furniture for the nidirect theme, once this and its corresponding PR are merged you can use the `site_search_block` to add to any locations it might be needed.

Reply:
Now done + pushed